app/validators/can_add_more_choices_validator.rb

```diff
@@ -3,5 +3,9 @@ def validate_each(record, attribute, application_choice)
     unless application_choice.application_form.can_submit_further_applications?
       record.errors.add(attribute, :max_course_choices, message: 'You cannot submit this application because you have already submitted the maximum number of applications.')
     end
+
+    if application_choice.application_form.exceeded_maximum_unsuccessful_choices?
+      record.errors.add(attribute, :max_course_choices, message: "You cannot submit this application because you have #{ApplicationForm::MAXIMUM_NUMBER_OF_UNSUCCESSFUL_APPLICATIONS} unsuccessful applications.")
```

Comment:
Technically, because you're using `>=` [here](https://github.com/DFE-Digital/apply-for-teacher-training/pull/8569/files#diff-22fdd3b57e65e63df6dd2ff571a3632cbb3a7a45f1ec11fbb3b338753fdb6c4bR298), the unsuccessful applications count is `count_unsuccessful_choices(ignore_inactive: true)` and not `ApplicationForm::MAXIMUM_NUMBER_OF_UNSUCCESSFUL_APPLICATIONS`.